modal/container_process.py

```diff
@@ -31,6 +34,8 @@ def __init__(
         client: _Client,
         stdout: StreamType = StreamType.PIPE,
         stderr: StreamType = StreamType.PIPE,
+        timeout_secs: Optional[int] = None,
+        exec_start: Optional[datetime] = None,
```

Comment:
Could these be combined into one field, `exec_deadline` which is expressed as a timestamp in the future?